devito/ir/clusters/analysis.py

```diff
@@ -237,7 +256,7 @@ def process(self, elements):
     def _callback(self, clusters, d, prefix):
         # A Dimension TILABLE only if it's PARALLEL and AFFINE
```

Comment:
Dimension `is`?

Reply:
good catch -- already fixed in the subsequent PR